UnitsNet/QuantityInfo.cs

```diff
@@ -203,7 +203,7 @@ public QuantityInfo(QuantityType quantityType, UnitInfo<TUnit>[] unitInfos, TUni
         /// <inheritdoc />
         public QuantityInfo(string name, UnitInfo<TUnit>[] unitInfos, TUnit baseUnit, IQuantity<TUnit> zero, BaseDimensions baseDimensions,
             QuantityType quantityType = QuantityType.Undefined)
-            : base(name, unitInfos, baseUnit, zero, baseDimensions, quantityType)
+            : base(name, unitInfos.Cast<UnitInfo>().ToArray(), baseUnit, zero, baseDimensions, quantityType)
```

Comment:
Not a big deal, but I think this can be simplified to `unitInfos.ToArray<UnitInfo>()`

Reply:
Fixed